### PR TITLE
Add JavaScript Question on Objects and Primitive values

### PR DESCRIPTION
**Pull request by khairalanam**
## Summary of changes

This PR adds a question to the JavaScript quiz on objects and primitive values.

## Checklist

- [x] I have read and followed the [contribution guidelines](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CONTRIBUTING.md).
- [x] I have read through the [Code of Conduct](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CODE_OF_CONDUCT.md) and agree to abide by the rules.
- [x] This PR is for one of the available issues and is not a PR for an issue already assigned to someone else.
- [x] My PR title has a short descriptive name so the maintainers can get an idea of what the PR is about.
- [x] I have provided a summary of my changes.

### CODE REVIEW

File: src/data/javascript-quiz.ts [hunk 1]
```diff
@@ -2052,6 +2052,17 @@ const javascriptQuiz = [
     Explanation:
       "In most web browsers, a single cookie can store up to 4 KB (kilobytes) of data.",
     Link: "https://www.freecodecamp.org/news/everything-you-need-to-know-about-cookies-for-web-development/"
+  },
+  {
+    Question:
+      "Which one of the following values is considered as a primitive value?",
```
Comment (jdwilkin4) [inline src/data/javascript-quiz.ts:2058]:
```suggestion
      "Which one of the following is considered a primitive value in JavaScript?",
```

File: src/data/javascript-quiz.ts [hunk 2]
```diff
@@ -2052,6 +2052,17 @@ const javascriptQuiz = [
...
+    Answer: "3.14",
+    Distractor1: "[1, 4, 9, 16]",
+    Distractor2: "function add(x, y) { return x + y; }",
+    Distractor3: "Date('2023-10-25')",
+    Explanation:
+      "There are 7 primitive data types in JavaScript; namely string, number, boolean, null, undefined, symbol, and bigint. Everything else is a JavaScript object including arrays, sets, functions, dates, etc.",
```
Comment (jdwilkin4) [inline src/data/javascript-quiz.ts:2064]:
```suggestion
      "There are 7 primitive data types in JavaScript: strings, numbers, booleans, null, undefined, Symbol, and BigInt. Everything else is a JavaScript object including arrays, sets, functions, dates, etc.",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by khairalanam** [comment]
I have implemented the changes.

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

**Comment by khairalanam** [comment]
Thank you!

**Pull request merged**